jupyter-lib/api/src/main/kotlin/org/jetbrains/kotlinx/jupyter/util/serializers.kt

```diff
@@ -21,6 +21,8 @@ import org.jetbrains.kotlinx.jupyter.api.libraries.ResourceFallbacksBundle
 import kotlin.reflect.KClass
 import kotlin.reflect.KProperty1
 
+val emptyJsonObject = JsonObject(mapOf())
```

Comment:
It pollutes the global namespace. Probably better to make it internal or hide behind `Json.Companion.EMPTY`

Reply:
Added `Json.EMPTY` extension intead